app/components/Home.tsx

```diff
@@ -0,0 +1,270 @@
+import * as React from 'react';
+const { EditorKit, EditorKitDelegate } = require('sn-editor-kit');
+const MarkdownIt = require('markdown-it');
+
+const EditMode = 0;
+const SplitMode = 1;
+const PreviewMode = 2;
+
+const modes = [
+  { mode: EditMode, label: "Edit", css: "edit" },
+  { mode: SplitMode, label: "Split", css: "split" },
+  { mode: PreviewMode, label: "Preview", css: "preview" },
+];
+
+const markdownitOptions = {
+  // automatically render raw links as anchors.
+  linkify: true
+};
+
+const MarkdownParser = MarkdownIt(markdownitOptions)
+  .use(require('markdown-it-footnote'))
+  .use(require('markdown-it-task-lists'))
+  .use(require('markdown-it-highlightjs'));
+
+type AppProps = {
+  text?: string,
+  mode: any;
+  platform?: any,
+}
+
+const initialState = {
+  mode: modes[0],
+  text: "",
+}
+const debugMode = false;
+
+if (debugMode) {
+  console.log("home loaded 2");
+}
+let keyMap = new Map();
+
+export class Home extends React.Component<{}, AppProps> {
+  editorKit: any;
+  note: any;
+
+  constructor(props: AppProps) {
+    super(props);
+    if (debugMode) {
+      console.log("started");
```

Comment:
These log statements are not helpful. They should either be deleted or edited to be less generic. A developer seeing this in the console would just see "started". What started?

Reply:
OK I'll remove them